apps/aecore/lib/aecore/naming/tx/name_transfer_tx.ex

```diff
@@ -44,8 +45,12 @@ defmodule Aecore.Naming.Tx.NameTransferTx do
   # Callbacks
 
   @spec init(payload()) :: t()
+  # TODO integrate Identifiers
   def init(%{hash: hash, target: target}) do
-    %NameTransferTx{hash: hash, target: target}
+    {:ok, identified_name_hash} = Identifier.create_identity(hash, :name)
+    {:ok, identified_target} = Identifier.create_identity(target, :account)
+    # TODO inspect target and its representation and adjust it
```

Comment:
is this todo still to be done?

Reply:
This is just a part of unused comments, will remove them soon.